packages/tabs/src/use-tabs.ts

```diff
@@ -449,7 +444,7 @@ export function useTabIndicator(): React.CSSProperties {
 
   return {
     position: "absolute",
-    transitionProperty: "left, right, top, bottom",
+    transitionProperty: "left, right, top, bottom, width",
```

Comment:
I guess `height` should be included here as well, in the case of vertical tabs. Right?

Reply:
Makes sense. I've included `height` now as well.